images/kubekins-test/Dockerfile-1.11

```diff
@@ -0,0 +1,48 @@
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
this is mostly a copy from the 1.10 file with a few fixes. dropped docker, fixed the maintainer label, and then switched the base image version